### DISCUSSION TITLE
Analyzing qPCR data

### DISCUSSION THREAD

**Discussion by sr320** [RobertsLab/resources#1467] [Q&A]
What is the most appropriate way to analyze qPCR data for gene expression. We have 1 target gene and 1 normalizing gene run on CFX connect machine.

**Comment by kubu4** [comment]
One "quick" way is to use the built in gene expression analysis aspect of the CFX Maestro software. It'll do the normalization and create plots; possibly also evaluate statistical significance? However, it's lacking in reproducibility to the proprietary nature of the software...

Just tossing it out there, but realize it's not ideal since people won't be able to do this analysis on their own without access to the software.

**Comment by kubu4** [comment]
But, this is a pretty nice, concise summary of using the delta delta Cq method for expression analysis:

https://horizondiscovery.com/-/media/Files/Horizon/resources/Technical-manuals/delta-cq-solaris-technote.pdf
